src/main/java/core/basesyntax/model/User.java

```diff
@@ -58,4 +64,12 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(login, password, age);
     }
+
+    @Override
+    public String toString() {
+        return "User{" + "id=" + id
+                + ", log='" + login
+                + "', pass='" + password
+                + "', age=" + age + '}';
+    }
```

Comment:
you don't need this method.

Reply:
oops, forgot to delete it. sorry, I was using it to check.